ext/hash/hash.stub.php

```diff
@@ -2,15 +2,15 @@
 
 /** @generate-function-entries */
 
-function hash(string $algo, string $data, bool $binary = false): string|false {}
+function hash(string $algo, string $data, bool $binary = false, array $args = []): string|false {}
```

Comment:
```suggestion
function hash(string $algo, string $data, bool $binary = false, array $options = []): string|false {}
```
This is what we usually call these kinds of arguments.

Reply:
Fixed, thanks.